BUILD

```diff
@@ -16,14 +16,15 @@
 #
 
 load("//:deployment.bzl", deployment_github = "deployment")
-load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_jvm_binary", "kt_jvm_library")
 load("@rules_pkg//:pkg.bzl", "pkg_zip")
 load("@vaticle_dependencies//distribution:deployment.bzl", "deployment")
 load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
 load("@vaticle_bazel_distribution//common:rules.bzl", "checksum", "assemble_targz", "assemble_zip", "java_deps", "assemble_versioned")
 load("@vaticle_bazel_distribution//common/tgz2zip:rules.bzl", "tgz2zip")
 load("@vaticle_bazel_distribution//github:rules.bzl", "deploy_github")
 load("@vaticle_bazel_distribution//brew:rules.bzl", "deploy_brew")
+load("@io_bazel_rules_kotlin//kotlin:jvm.bzl", "kt_jvm_binary", "kt_jvm_library")
+load("@io_bazel_rules_kotlin//kotlin:core.bzl", "kt_compiler_plugin")
```

Comment:
We don't import all our methods through `kotlin:kotlin.bzl` anymore, though the methods on the file allow you to they advise that we use the appropriate file.

```
def kt_jvm_library(**kwargs):
    """
    Forwarding macro for kt_jvm_library

    Deprecated:
        kt_jvm_library should be loaded from //kotlin:jvm.bzl
    """
    print("kt_jvm_library should be loaded from //kotlin:jvm.bzl")
    _kt_jvm_library(**kwargs)
```